cfe_internal/enterprise/federation/federation.cf

```diff
@@ -212,6 +212,11 @@ bundle agent federation_manage_files
     "feeder_username" data => parsejson('{"feeder_username":"$(cfengine_enterprise_federation:config.transport_user)"}');
     "feeder" data => parsejson('{"feeder": "$(sys.key_digest)"}');
 
+    "DEBUG|DEBUG_$(this.namespace)_$(this.bundle)"::
+      "debug" data => parsejson('{ "classes": { "DEBUG":true}}');
+    "!(DEBUG|DEBUG_$(this.namespace)_$(this.bundle))"::
+      "debug" data => parsejson('{ "classes": { "DEBUG":false}}');
```

Comment:
I don't like this `classes.DEBUG`, let's separate the bash stuff from the policy and keep the bash stuff as simple as possible. IOW, just `{"debug": "true/false"}`, also having it as a string, not a JSON boolean.

Reply:
yeah, I don't really care, was just rolling with what @nickanderson 's suggestion :)